packages/common/lib/index.ts

```diff
@@ -1,9 +1,9 @@
 import { Loggers } from '@sphereon/ssi-types';
 
 export const VCI_LOGGERS = Loggers.DEFAULT;
-export const VCI_LOG_COMMON = VCI_LOGGERS.get('sphereon:oid4vci:common');
+export const VCI_LOG_COMMON = VCI_LOGGERS.get('sphereon:common');
```

Comment:
Please leave 'oid4vc' in there. We use these namespaces in our DEBUG logging to be able to selectively log certain namespaces. Now it is being moved up to the sphereon namspace